ush/python/pygfs/task/aero_analysis.py

```diff
@@ -0,0 +1,238 @@
+from pygfs.task.analysis import Analysis
+from pygw.file_utils import FileHandler
+from pygw.fsutils import rm_p
+from pygw.template import Template, TemplateConstants
+from pygw.yaml_file import YAMLFile
+import datetime as dt
+import glob
+import gzip
+import netCDF4 as nc
+import os
+import tarfile
+
+
+class AerosolAnalysis(Analysis):
+    """
+    Class for global aerosol analysis tasks
+    """
+    def __init__(self, config):
+        super().__init__(config)
+
+    def initialize(self):
+        super().initialize()
+        # stage fix files
+        crtm_fix_list_path = os.path.join(self.config['HOMEgfs'], 'parm', 'parm_gdas', 'aero_crtm_coeff.yaml')
+        crtm_fix_list = YAMLFile(path=crtm_fix_list_path)
+        crtm_fix_list = Template.substitute_structure(crtm_fix_list, TemplateConstants.DOLLAR_PARENTHESES, self.runtime_config.get)
+        FileHandler(crtm_fix_list).sync()
+        jedi_fix_list_path = os.path.join(self.config['HOMEgfs'], 'parm', 'parm_gdas', 'aero_jedi_fix.yaml')
+        jedi_fix_list = YAMLFile(path=jedi_fix_list_path)
+        jedi_fix_list = Template.substitute_structure(jedi_fix_list, TemplateConstants.DOLLAR_PARENTHESES, self.runtime_config.get)
+        FileHandler(jedi_fix_list).sync()
+
+        # stage berror files
+        if self.config.get('STATICB_TYPE', 'bump_aero') in ['bump_aero']:
+            # copy BUMP files, otherwise it will assume ID matrix
+            FileHandler(self.get_berror_dict()).sync()
+
+        # stage backgrounds
+        FileHandler(self.get_bkg_dict()).sync()
+
+        # generate variational YAML file
+        yaml_out = os.path.join(self.config['DATA'], f"{self.config['CDUMP']}.t{self.cyc}z.aerovar.yaml")
+        varda_yaml = YAMLFile(path=self.config['AEROVARYAML'])
+        varda_yaml = Template.substitute_structure(varda_yaml, TemplateConstants.DOUBLE_CURLY_BRACES, self.runtime_config.get)
+        varda_yaml = Template.substitute_structure(varda_yaml, TemplateConstants.DOLLAR_PARENTHESES, self.runtime_config.get)
+        varda_yaml.save(yaml_out)
+
+        # link var executable
+        exe_src = self.config['JEDIVAREXE']
+        exe_dest = os.path.join(self.config['DATA'], os.path.basename(exe_src))
+        if os.path.exists(exe_dest):
+            rm_p(exe_dest)
+        os.symlink(exe_src, exe_dest)
```

Comment:
I'll need to check if it is still the case, but JEDI executables were a bit finicky with copy vs link because of shared libraries (JEDI is built as a series of libraries and then executables are built from those libraries). Maybe @aerorahul has a suggestion? Would `cp` + `LD_LIBRARY_PATH` work?

Reply:
Concur with @CoryMartin-NOAA here, you can't simply copy the executable.

Reply:
Fine for now, but down the line that is going to cause a problem with NCO.

Reply:
Oh for sure, it needs fixed, just hasn't been done yet